src/sidebar/components/Annotation/test/EmptyAnnotation-test.js

```diff
@@ -0,0 +1,145 @@
+import { mount } from 'enzyme';
+
+import { checkAccessibility } from '../../../../test-util/accessibility';
+import { mockImportedComponents } from '../../../../test-util/mock-imported-components';
+
+import EmptyAnnotation, { $imports } from '../EmptyAnnotation';
+
+describe('EmptyAnnotation', () => {
+  let fakeOnToggleReplies;
+
+  const createComponent = props => {
+    return mount(
+      <EmptyAnnotation
+        hasAppliedFilter={false}
+        isReply={false}
+        onToggleReplies={fakeOnToggleReplies}
+        replyCount={0}
+        threadIsCollapsed={true}
+        {...props}
+      />
+    );
+  };
+
+  beforeEach(() => {
+    fakeOnToggleReplies = sinon.stub();
+
+    $imports.$mock(mockImportedComponents());
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
+
+  describe('reply thread toggle', () => {
+    it('should render a toggle button if the annotation has replies', () => {
```

Comment:
Is the logic here the same as for Annotation? Could it be pushed down into `AnnotationReplyToggle`?